Cue/app/api/types.js

```diff
@@ -14,6 +14,7 @@ export type DeckMetadata = {
   author: string,
   rating: number,
   num_ratings: number,
+  num_cards: number,
```

Comment:
While you're in api/types.js, can you also take a look at #109? If you don't want to fiddle with inheritance in Flow, I'm ok with just copy/pasting DeckMetadata into Deck and adding a reminder comment to keep them synchronized.